wp-content/mu-plugins/register-taxonomies.php

```diff
@@ -314,7 +314,7 @@
    *
```

Comment:
Taxonomy terms for employer programs, for use in filtering through the archive scripts